bundle-ocata-s390x.yaml

```diff
@@ -190,7 +190,7 @@ services:
     num_units: 1
     options:
       worker-multiplier: 0.5
-      bridge-mappings: physnet1:br-ex
+      ext-port: eth1
```

Comment:
Ack. I've also not been able to make the new-style network expression work with the openstack-on-lxd scenarios.

@javacruft - do we need to keep using the old-style ext-port approach for this scenario?  Can you think of any reasons the new style wouldn't work?